## This combines multiple opcodes
uint8 modecode
## 0 is Power (opcode 133) ; puts roomba to sleep
## 1 is Start (opcode 128) ; Starts SCI in passive mode
## 2 is Safe (opcodes 131 & 130) ; Puts the SCI in safe mode
## 3 is Full  (Opcode 132) ; puts the SCI in Full mode
